### ISSUE TITLE
Sublime Text should be restarted, otherwise one or more of the installed packages may not function properly

### ISSUE THREAD

**Issue by nodesocket** [wbond/package_control#994]
Getting:

```
1 missing dependency was just installed. Sublime Text should be restarted, otherwise one or more of the installed packages may not function properly.
```

On open, even though I've restarted Sublime dozens of times. Opening up the console reveals:

```
Package Control: Installed missing dependency bz2
Package Control: Skipping automatic upgrade, last run at 2015-09-15 17:44:34, next run at 2015-09-15 18:44:34 or after
error: Package Control

1 missing dependency was just installed. Sublime Text should be restarted, otherwise one or more of the installed packages may not function properly.
```

**Comment by wbond** [comment]
Seems to be a duplicate of #989. Can you execute the command from the comment on that issue and post your results there?

**Comment by nodesocket** [comment]
I ran the following command on my Mac, but still seeing the error:

````
unzip -l ~/Library/Application\ Support/Sublime\ Text\ 3/Installed\ Packages/0_package_control_loader.sublime-package
```

**Comment by wbond** [comment]
@nodesocket What was the output?

**Comment by nodesocket** [comment]
````
➜  ~  unzip -l ~/Library/Application\ Support/Sublime\ Text\ 3/Installed\ Packages/0_package_control_loader.sublime-package
Archive:  /Users/justin/Library/Application Support/Sublime Text 3/Installed Packages/0_package_control_loader.sublime-package
  Length     Date   Time    Name
 --------    ----   ----    ----
      173  07-21-15 17:47   dependency-metadata.json
     1877  07-21-15 17:47   00-package_control.py
       69  07-21-15 17:47   02-bz2.py
       69  07-22-15 20:54   02-bz2.py
       68  09-14-15 10:37   02-bz2.py
       68  09-14-15 10:37   02-bz2.py
       68  09-14-15 10:38   02-bz2.py
       68  09-14-15 11:43   02-bz2.py
       68  09-14-15 13:55   02-bz2.py
       68  09-14-15 15:53   02-bz2.py
       68  09-14-15 17:04   02-bz2.py
       68  09-14-15 17:20   02-bz2.py
       68  09-14-15 17:42   02-bz2.py
       68  09-14-15 18:02   02-bz2.py
       68  09-15-15 11:31   02-bz2.py
       68  09-15-15 11:31   02-bz2.py
       68  09-15-15 11:32   02-bz2.py
       68  09-15-15 11:34   02-bz2.py
       68  09-15-15 11:34   02-bz2.py
       68  09-15-15 11:35   02-bz2.py
       68  09-15-15 13:24   02-bz2.py
       68  09-15-15 13:24   02-bz2.py
       68  09-15-15 13:29   02-bz2.py
       68  09-15-15 17:44   02-bz2.py
       68  09-15-15 17:45   02-bz2.py
       68  09-15-15 17:48   02-bz2.py
       68  09-15-15 17:49   02-bz2.py
       68  09-15-15 17:50   02-bz2.py
       68  09-15-15 17:51   02-bz2.py
       68  09-15-15 17:52   02-bz2.py
       68  09-16-15 10:13   02-bz2.py
       68  09-16-15 12:27   02-bz2.py
       68  09-16-15 12:57   02-bz2.py
       68  09-16-15 13:00   02-bz2.py
 --------                   -------
     4228                   34 files
````

**Comment by wbond** [comment]
Can you paste the rest of your Console output upon a fresh start?

**Comment by nodesocket** [comment]
````
startup, version: 3083 osx x64 channel: stable
executable: /Applications/Sublime Text.app/Contents/MacOS/Sublime Text
working dir: /
packages path: /Users/justin/Library/Application Support/Sublime Text 3/Packages
state path: /Users/justin/Library/Application Support/Sublime Text 3/Local
zip path: /Applications/Sublime Text.app/Contents/MacOS/Packages
zip path: /Users/justin/Library/Application Support/Sublime Text 3/Installed Packages
ignored_packages: ["Markdown", "MarkdownEditing", "Vintage"]
pre session restore time: 0.26154
using gpu buffer for window
first paint time: 0.324735
startup time: 0.344903
reloading plugin Default.block
reloading plugin Default.comment
reloading plugin Default.copy_path
reloading plugin Default.delete_word
reloading plugin Default.detect_indentation
reloading plugin Default.duplicate_line
reloading plugin Default.echo
reloading plugin Default.exec
reloading plugin Default.fold
reloading plugin Default.font
reloading plugin Default.goto_line
reloading plugin Default.history_list
reloading plugin Default.indentation
reloading plugin Default.kill_ring
reloading plugin Default.mark
reloading plugin Default.new_templates
reloading plugin Default.open_context_url
reloading plugin Default.open_file_settings
reloading plugin Default.open_in_browser
reloading plugin Default.pane
reloading plugin Default.paragraph
reloading plugin Default.paste_from_history
reloading plugin Default.quick_panel
reloading plugin Default.save_on_focus_lost
reloading plugin Default.scroll
reloading plugin Default.set_unsaved_view_name
reloading plugin Default.side_bar
reloading plugin Default.sort
reloading plugin Default.swap_line
reloading plugin Default.switch_file
reloading plugin Default.symbol
reloading plugin Default.transform
reloading plugin Default.transpose
reloading plugin Default.trim_trailing_white_space
reloading plugin CSS.css_completions
reloading plugin Diff.diff
reloading plugin HTML.encode_html_entities
reloading plugin HTML.html_completions
reloading plugin 0_package_control_loader.00-package_control
reloading plugin 0_package_control_loader.02-bz2
reloading plugin GitGutter.git_gutter
reloading plugin GitGutter.git_gutter_change
reloading plugin GitGutter.git_gutter_compare
reloading plugin GitGutter.git_gutter_events
reloading plugin GitGutter.git_gutter_handler
reloading plugin GitGutter.git_helper
reloading plugin GitGutter.view_collection
reloading plugin LESS.less_completions
reloading plugin Package Control.1_reloader
reloading plugin Package Control.2_bootstrap
reloading plugin Package Control.Package Control
reloading plugin Pretty JSON.PrettyJson
[Errno 2] No such file or directory: 'jq'
reloading plugin SublimeLinter-annotations.linter
SublimeLinter: annotations linter loaded 
reloading plugin SublimeLinter-coffee.linter
SublimeLinter: coffee linter loaded 
reloading plugin SublimeLinter-csslint.linter
SublimeLinter: csslint linter loaded 
reloading plugin SublimeLinter-jshint.linter
SublimeLinter: jshint linter loaded 
reloading plugin SublimeLinter-json.linter
SublimeLinter: json linter loaded 
reloading plugin SublimeLinter-php.linter
SublimeLinter: php linter loaded 
reloading plugin SublimeLinter-ruby.linter
SublimeLinter: ruby linter loaded 
reloading plugin SublimeLinter-shellcheck.linter
SublimeLinter: shellcheck linter loaded 
reloading plugin JSHint.JSHint
reloading plugin SublimeLinter.commands
reloading plugin SublimeLinter.sublimelinter
plugins loaded
SublimeLinter: debug mode: off 
SublimeLinter: annotations activated: <builtin> 
Package Control: Installing 1 missing dependency
Package Control: Installed missing dependency bz2
Package Control: No updated packages
error: Package Control

1 missing dependency was just installed. Sublime Text should be restarted, otherwise one or more of the installed packages may not function properly.
````

**Comment by nodesocket** [comment]
@wbond any updates?

**Comment by OrKoN** [comment]
I have the same issue. PackageControl keeps installing 1 missing dependency on each restart

**Comment by wbond** [comment]
This issue was closed indicating that it is a duplicate of #989, meaning there will not be any more updates here.
